### DISCUSSION TITLE
Question about HW2, the motion of body

### DISCUSSION THREAD

**Discussion by Maid-Unknown** [cooperrc/advanced-dynamics#28] [General]
In hw2, the lenth of each bar is 1. Thus, When the bar moves to vertical position (90'), the bar will overlap and the postion of C will be (0,0). However, if we continue the rotation. point C will have two move way, either go to left side or back to right side. Which type of motion I should consider?

**Comment by Maid-Unknown** [comment]
also, the plot is that means have to be like a small video? or just normal 2D plot graph

**Reply by travismalave** [reply]
I think its a normal, non-animated plot based on his comments on animation from HW 01.
